app/models/bridge.rb

```diff
@@ -33,6 +33,9 @@ class Bridge < ApplicationRecord
   validates :delay, inclusion: DELAYS
   validates :retries, inclusion: RETRIES
 
+  before_validation :set_inbound_url, only: :inbound_url
```

Comment:
This should be a before_create. Currently you will overwrite the inbound_url on every validation. If you do bridge.valid?, it's overwritten. bridge.update, overwritten. We want this to run once which is right before it creates.